api.js

```diff
@@ -429,17 +429,17 @@ module.exports = async function attachAPI(app, {wss, db}) {
         const filename = `${basename}.${ext}`
 
         req[middleware.vars].path += '/' + filename
-
         cb(null, filename)
       }
     })
   })
   const uploadSingleImage = upload.single('image')
 
+  // TODO: delete old images
```

Comment:
This should be a server option. Also, custom emoji images should *not* be deleted, which will be something to consider.

Reply:
...should this be added later?